### ISSUE TITLE
Collectives can't apply to host on legacy plan because of 25 collectives limit 

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#2913]
Our host https://opencollective.com/allforclimate was created last year and is under the legacy plan which means that there shouldn't be any limit. However, when people apply to host their collective on that host they get the error message that we have reached the limit of collectives we can host under our plan (25).

**Comment by znarf** [comment]
Hosts were assigned a legacy plan (small, medium, large) that was matching their current usage. So, this is working as expected.

We may tweak the plan for this specific host. cc @piamancini

**Comment by xdamman** [comment]
I see in the database that the plan is set to `legacy-large-host-plan` so something must be off.

**Comment by znarf** [comment]
Large is up to 25 collectives so that is perfectly on point.

**Comment by xdamman** [comment]
Ah I didn’t know. 25 doesn’t seem like a “large” number. I also didn’t know that the legacy plan was restricting old hosts to their current usage and prevented them to grow. That wasn’t clear in the communication.

In any case thank you for giving me those precisions during the weekend!

**Comment by piamancini** [comment]
This what we said in the email we sent, I think it was communicated.  

<img width="579" alt="Screen Shot 2020-02-16 at 7 38 27 PM" src="https://user-images.githubusercontent.com/3671070/74610491-f7a0b180-50f3-11ea-9d75-315839d185d8.png">

25 was what we agreed was the top level, there are few hosts over that. We can certainly look at upping the limit but again, we don't have that many hosts with +25 collectives (other than ours). We can't also keep changing the pricing model based on how many collectives this host has. What would make sense as a large host?

**Comment by xdamman** [comment]
Ah ok. I missed that. Indeed I wouldn’t change this. At least not now.

AllForClimate falls in a different category though as it is a host that I created for the climate vertical. So I wouldn’t impose any such restriction on it in 2020 and then we can reconsider. Right now there is a 0% host fee so there isn’t any budget available. But I’m sure that this will change within the year. Working on it. But right now some local chapters of #FridaysForFuture are waiting to be able to join that host.

**Comment by alanna** [comment]
So will any host that does not charge host fees to its Collectives be able to have an unlimited account for free? That doesn't seem right to me @xdamman - we need large hosts to help support our sustainability, even if they are still working on their own. What about applying for a grant to help cover operational costs for AllForClimate, including the OC fees?

**Comment by xdamman** [comment]
> So will any host that does not charge host fees to its Collectives be able to have an unlimited account for free?

Of course not. That’s not what I wrote. 

> we need large hosts to help support our sustainability

Absolutely

> What about applying for a grant to help cover operational costs for AllForClimate, including the OC fees?

Absolutely. 

So in the meantime we say no to new climate collectives that are waiting to apply to that host?

**Comment by alanna** [comment]
> Of course not. That’s not what I wrote.

Well a lot of companies have a nonprofit discount deal so I thought maybe we should consider a policy about it. Maybe we should use AllForClimate as a test case for that.

> So in the meantime we say no to new climate collectives that are waiting to apply to that host?

I'll leave that to @piamancini to make the call. Above 25 Collectives we've agreed to do custom deals with hosts, but we haven't actually created guidelines for that yet.

**Comment by xdamman** [comment]
(Note: technically most of the fiscal hosts are not for profit (e.g. wwcode), so I’d be cautious with such a discount)

**Comment by piamancini** [comment]
It seems like an edge case, since we created the host ourselves. It was done by @xdamman while he was still full time on OC. I would treat it like Foundation / OSC. With the difference that in this case no fees are paid at all for added funds. 

I think we should place them on a larger legacy plan for 2020 to a custom plan on the understanding of fairness, if the host grows to be able to cover those fees, it should get on a paid plan.

**Comment by piamancini** [comment]
Host is unblocked. Closing the issue.
